src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -1,17 +1,32 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
-// Simplify relationship to a single key/value.
-function transformRelationship(relationship) {
-  let ret;
-  if (typeof relationship === 'string') {
-    ret = relationship; // Nothing to do here
-  } else if (relationship?.relationshipToVeteran) {
-    // If they chose "other" and typed a freeform answer, grab the text
-    ret = relationship.otherRelationshipToVeteran
-      ? relationship.otherRelationshipToVeteran
-      : relationship.relationshipToVeteran;
+// Rearranges date string from YYYY-MM-DD to MM-DD-YYYY
+function fmtDate(date) {
+  return date?.length && date.length === 10
+    ? `${date.slice(5)}-${date.slice(0, 4)}`
+    : date;
+}
+
+// Simplify a relationship object from (potentially) multiple nested objects to a single string
+function transformRelationship(obj) {
+  if (typeof obj === 'string') return obj;
+  let rel = obj?.relationshipToVeteran;
+  // Check if relationshipToVeteran is an object
+  if (typeof obj?.relationshipToVeteran === 'object') {
+    // Iterate over keys in relationshipToVeteran object
+    Object.keys(obj?.relationshipToVeteran).forEach(key => {
+      // edge case for certifier relationship (generated by a checkboxgroup):
+      if (obj?.relationshipToVeteran[key] === true) {
+        rel = key; // e.g., `'spouse': true` yields `'spouse'`
+      }
+    });
   }
-  return ret || '';
+  // Check if otherRelationshipToVeteran is a string
+  if (typeof obj?.otherRelationshipToVeteran === 'string') {
+    rel = obj?.otherRelationshipToVeteran;
+  }
+
+  return rel;
 }
```

Comment:
Had to rework this function because the certifier relationship and applicant relationship produce varied outputs. So sometimes we might have a value like:
```javascript
// When relationship comes from a checkbox group:
{
  relationshipToVeteran: {other: false, parent: true}
}
```
or,
```javascript
// When relationship comes from a standard radio
{
  relationshipToVeteran: 'spouse'
}
```
or,
```javascript
// When relationship comes from a radio with the `other` value selected
{
  relationshipToVeteran: 'other',
  otherRelationshipToVeteran: 'uncle'
}
```
depending on where in the form this relationship is collected